src/components/layout/UserSettings/index.tsx

```diff
@@ -6,127 +6,99 @@ import { WalletButton } from 'components/common/WalletButton';
-                    rel="noreferrer"
-                  >
-                    <div>{t('Docs')}</div>
-                    <FontIcon
-                      name={FontIconName.Book} 
-                      size={16}
-                    />
-                  </a>
-                </li>
-                <li className={styles.paper}>
-                  <a
-                    className={styles.head}
-                    href={`${process.env.PUBLIC_URL}/RicochetExchangeWhitepaper.pdf`}
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    <div>{t('White Paper')}</div>
-                    <FontIcon
-                      name={FontIconName.Paper} 
-                      size={16}
-                    />
-                  </a>
-                </li>
-                <li className={styles.discord}>
-                  <a
-                    className={styles.head}
-                    href="https://discord.gg/mss4t2ED3y"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    <div>{t('Discord')}</div>
-                    <FontIcon
-                      name={FontIconName.Chat}
-                      size={16}
-                    />
-                  </a>
-                </li>
-                <li className={styles.github}>
-                  <a
-                    className={styles.head}
-                    href="https://github.com/Ricochet-Exchange"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    <div>{t('GitHub')}</div>
-                    <FontIcon
-                      name={FontIconName.GitHub}
-                      size={16}
-                    />
-                  </a>
-                </li>
-                <li className={styles.discord}>
-                  <a
-                    className={styles.head}
-                    href="https://twitter.com/ricochetxchange"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    <div>{t('Twitter')}</div>
-                    <FontIcon
-                      name={FontIconName.Twitter}
-                      size={16}
-                    />
-                  </a>
-                </li>
-                <li className={styles.terms}>
-                  <a
-                    className={styles.head}
-                    href="https://github.com/Ricochet-Exchange/ricochet-frontend/blob/main/TERMS.md"
-                    target="_blank"
-                    rel="noreferrer"
-                  >
-                    <div>{t('Terms')}</div>
-                    <FontIcon
-                      name={FontIconName.Lock}
-                      size={16}
-                    />
-                  </a>
-                </li>
-              </ul>
-            </div>
-          </Dropdown>
-        </div>
-      </div>
-    </div>
-  );
```

Comment:
There was too much changed in this file for me to see what was changed. It looks like your editor automatically fixed a bunch of whitespace. In the future, don't create a commit diff like this if you can, commits should only edit the exact lines that are within the scope of the issue. A commit like this looks sloppy, I understand the whitespace isn't consistent through out the app, so I don't think you need to change anything here. Just keep this in mind going forward, it is a secure software development practice to only edit the lines that need editing, you can see how I may potentially miss an error because of how much was changed here.